src/nnet3bin/nnet3-egs-augment-image.cc

```diff
@@ -231,6 +241,14 @@ void PerturbImage(const ImageAugmentationConfig &config,
   // [ cos(theta)  -sin(theta)  0
   //   sin(theta)  cos(theta)   0
   //   0           0            1 ]
+  if (RandUniform() <= config.rotation_prob) {
```

Comment:
I guess you could use Kaldi's `WithProb()` which makes it nicer.
Not important though.